__tests__/main.test.ts

```diff
@@ -120,6 +120,48 @@ describe('test npm-setup-publish', () => {
   })
 
   describe('update npmrc', () => {
+    test('Updates multiline npmrc without comments', async () => {
+      const repository = path.join(runnerTempDir as string, 'repo')
+      const npmrcPath = path.join(repository, '.npmrc')
+      await fs.mkdir(repository, {recursive: true})
+      process.chdir(repository)
+      await fs.writeFile(npmrcPath, '')
+      await fs.writeFile('package.json', '{}')
+
+      const newNpmrcContent = `
+registry = https://artifactor.ee/registry
+# test comment
+  # another comment = something
+email = test@example.com # more comments
```

Comment:
Are there any cases of usage with quotes?

```
key = "test@example.com # more comments"
```

Reply:
It looks like there are no usages of comments except [already removed in a branch](https://github.com/smartlyio/frontend/pull/19831/commits/66639864d59947ef83eb17f7918182b29cac9f32), and it was only whole line comments